main.cpp

```diff
@@ -281,15 +279,14 @@ float Daw::outputAudio(bool)
 {
     if(stereoOutput == true)
     {
-        std::cout<<"audio playing"<<std::endl;
+        return(audioTracks);
     }
     else
     {
-        std::cout<<"no audio"<<std::endl;
+        return 0;
     }
```

Comment:
a couple things here.
`return` is the last thing that happens in a function.   the `else` block is not needed because nothing else happens after the `else{ }` block's return statement.  If the `if() { }` body is not entered, it will be skipped.
```
if( expr )
    return ...;

return 0;
```
- [ ] remove `else` keyword.
- [ ] write `return audioTracks;` not `return(audioTracks);`